<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplace</class>
 <widget class="QDialog" name="FindReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find and Replace</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditReplace"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelReplace">
        <property name="text">
         <string>Replace:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditFind"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFind">
        <property name="text">
         <string>Find:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonsLayout">
      <item>
       <widget class="QPushButton" name="replaceNextButton">
        <property name="text">
         <string>Replace Next</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="replaceAllButton">
        <property name="text">
         <string>Replace All</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="optionsLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QCheckBox" name="wholeWordCheck">
        <property name="text">
         <string>Match Whole Word</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="caseSensitiveCheck">
        <property name="text">
         <string>Case Sensitive</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="searchKeyReplaced">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditFind</tabstop>
  <tabstop>lineEditReplace</tabstop>
  <tabstop>replaceNextButton</tabstop>
  <tabstop>replaceAllButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
